docs/src/pages/[platform]/getting-started/create-react-app/examples/BasicExampleCode.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, Flex, Heading, Image, Text } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+function App() {
+  return (
+    <Flex width="32rem" padding="2rem">
+      <Image
+        alt="Abstract painting"
+        width="10rem"
+        height="15rem"
+        src="https://images.unsplash.com/photo-1500462918059-b1a0cb512f1d?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987"
```

Comment:
Are there any copyright concerns we should be aware of with using images from `[Unsplash](https://unsplash.com/)`?

Reply:
It might be a good practice to include an attribution link. Related: I wonder if we should have a media page we put this kind of stuff on that we link to from the footer. The same page could house disclaimers for logo usage. We currently have a disclaimer for Flutter, but Figma has [similar guidelines](https://www.figma.com/using-the-figma-brand/) around their logo usage.

Reply:
+1 to Heather. Also, doesnt seem like any concern for using Unspalsh images. They say its freely usable as per https://unsplash.com/license.